src/columnsTypes/types/Label/Component.jsx

```diff
@@ -56,7 +45,7 @@ const LabelOption = ({
         onChange(title, !selected);
       }}
     >
-      <img
+      <span
```

Comment:
why?

Reply:
it was not working with the svg for some reason